libs/platform/kernel/ebpf_platform_kernel.c

```diff
@@ -37,7 +37,10 @@ ebpf_platform_terminate()
 __drv_allocatesMem(Mem) _Must_inspect_result_ _Ret_maybenull_
     _Post_writable_byte_size_(size) void* ebpf_allocate(size_t size)
 {
-    return ExAllocatePool2(POOL_FLAG_NON_PAGED, size, EBPF_POOL_TAG);
+    void* p = ExAllocatePoolZero(NonPagedPoolNx, size, EBPF_POOL_TAG);
+    if (p)
```

Comment:
redundant per: https://docs.microsoft.com/en-us/windows-hardware/drivers/ddi/wdm/nf-wdm-exallocatepoolzero

Reply:
The documentation is wrong. Failure to zero the memory results in bugcheck.

Reply:
There's a submit/view feedback section at the bottom of that page.  If it's wrong, please start a bug/discussion there.

Reply:
This is a known issue, as indicated on that page:
![image](https://user-images.githubusercontent.com/20480683/132869833-db4005e8-057c-49ab-bdfd-4b90ba4f4e5b.png)

Reply:
Since ebpf_allocate() can be called frequently, is there a version ifdef we can wrap it in so that we only zero on the version with the bug?   Would be good to not take this perf hit on non buggy versions if possible.

Reply:
Makes sense. Switching to the non-buggy API and explicitly zeroing the memory.